server/apps/api/urls.py

```diff
@@ -5,7 +5,7 @@
 from server.apps.api import views
 
 urlpatterns = [
-    path('domain/', views.DomainCreateAPIView.as_view(), name='create-domain'),
+    path('case/', views.CaseCreateAPIView.as_view(), name='create-case'),
     path('domains/', views.DomainListView.as_view(), name='domain-list'),
```

Comment:
Better to use underscore in names: `create_case`. In this case, you can copy the name when you need it with a double click.